types/v56/types.go

```diff
@@ -367,7 +367,7 @@ type Vdc struct {
 	ID           string `xml:"id,attr,omitempty"`
 	OperationKey string `xml:"operationKey,attr,omitempty"`
 	Name         string `xml:"name,attr"`
-	Status       string `xml:"status,attr,omitempty"`
+	Status       int    `xml:"status,attr,omitempty"`
```

Comment:
I think this one change is not for this PR :)

Reply:
Ya this change is based on my other PR https://github.com/vmware/go-vcloud-director/pull/198#pullrequestreview-248605514 I can revert it to the current master cause im not sure when that one will get merged in.

Reply:
I have revert this back for this pull request.